### DISCUSSION TITLE
<title>Changing back to github's default url domain

### DISCUSSION THREAD

**Discussion by Alif829** [community/community#73412] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I previously added a custom domain on a different repo. But as I'm deploying a new repository, it is fetching that custom domain from the other project. I have tried creating new repositories and deploying them. But the result is same. example link is- [](http://lutfaralif.xyz/test/), but I want to publish to default github url like {username}.github.io/test

**Comment by GigaSmurf** [comment]
Ensure that the custom domain field in your Pages setting in your repo settings are correct (or empty). Also, check your DNS settings with your domain provider that you change which repo you want to use. It should take a minute. Remember to check CNAME as well.
